r/tests/testthat/test-filesystem.R

```diff
@@ -130,6 +129,21 @@ test_that("LocalFileSystem + Selector", {
   expect_equal(sum(types == FileType$Directory), 1L)
 })
 
+test_that("clean_path_abs() marks its output as utf-8", {
+  f <- tempfile(fileext = "Público")
+  on.exit(unlink(f))
+
+  file.create(f)
+  f_utf8 <- enc2utf8(normalizePath(f, winslash = "/"))
+  f_latin1 <- iconv(f_utf8, "utf-8", "latin1")
+
+  expect_identical(clean_path_abs(f_utf8), f_utf8)
```

Comment:
Just to check, is it intended that this test passes with or without the change made in this PR, as does the similar one below?

Reply:
Nah, the second ones in those groups fail for me too on Ubuntu, it was just that particular expectation I was asking about.